lbz/resource.py

```diff
@@ -154,13 +159,10 @@ def __init__(
     ):
         # TODO: adjust the rest of the arguments in the near future too.
         super().__init__(event)
-        cors_headers = cors_headers or []
-        if not cors_headers and (env_headers := env.get("CORS_HEADERS")):
-            cors_headers = env_headers.split(",")
+        if not cors_headers:
+            cors_headers = cast(list, CORS_HEADERS.value)
```

Comment:
Here we have the same problem - it is very possible that we should not use lambda functions as parsers 🤔